7_3.py

```diff
@@ -0,0 +1,17 @@
+
+class Message:
+    """
+    A Message Representation Class
+
+    """
+    def __init__(self, title, body, sender, receiver):

```

Comment:
You should be using type hinting here.
https://stackoverflow.com/questions/44959876/type-hints-convention-for-instance-variables-python